unlock-app/src/components/content/certification/LinkedInShareButton.tsx

```diff
@@ -21,7 +21,14 @@ export const LinkedinShareButton = ({
   if (!metadata.name || !certificateData) {
     return null
   }
-  const certificationUrl = `${window?.location?.origin}/certification?lockAddress=${lockAddress}&network=${network}&tokenId=${tokenId}`
+
+  const slug = metadata.slug
+  let certificationUrl = ''
+  if (slug) {
+    certificationUrl = `${window?.location?.origin}/certification?s=${slug}`
+  } else {
+    certificationUrl = `${window?.location?.origin}/certification?lockAddress=${lockAddress}&network=${network}&tokenId=${tokenId}`
```

Comment:
Let's create a helper function `certificationUrl` and `eventUrl` that takes `metadata` so we don't have to always do this?